src/app/search/components/lists/list.module.css

```
@@ -0,0 +1,72 @@
+.lists {
+  width: 24.5rem;
+  padding: 10px;
+
+  li {
+    display: grid;
+    width: 100%;
+    height: 11rem;
+    gap: 1rem;
+    padding: 1rem;
+    border-bottom: 1px solid var(--Neutral_Shade-90);
+
+    &:hover {
+      border: 1px solid var(--Neutral_Shade-70);
+      border-radius: var(--border-radius2);
+    }
```

بهتره رو حالت عادی هم border داشته باشه که وقتی hover میکنیم جابه‌جا نشه

```suggestion
    border: 1px solid transparent;
    border-bottom-color: var(--Neutral_Shade-90);

    &:hover {
      border-color: var(--Neutral_Shade-70);
      border-radius: var(--border-radius2);
    }
```

---

مدل خود طرح اینجوری بود

---

اصلاح شد